plugins/keysend.c

```diff
@@ -3,6 +3,7 @@
 #include <wire/gen_onion_wire.h>
 
 #define PREIMAGE_TLV_TYPE 5482373484
+#define KEYSEND_FEATUREBIT 0x81
```

Comment:
Why isn't it [55](https://github.com/lightningnetwork/lightning-rfc/issues/605#issuecomment-606679798) ?

Reply:
I presume it's because 'bit 55' corresponds to 0x81 in hex.

Reply:
.... that doesn't make sense, @niftynei ?  0x81 == 129.  I assume this was supposed to be 0x37?